Sources/SwiftGRPCNIO/GRPCWebCorsHandler.swift

```diff
@@ -0,0 +1,73 @@
+import NIO
+import NIOHTTP1
+
+/// Handler that manages the CORS protocol for requests incoming from the browser.
+class GRPCWebCorsHandler {
```

Comment:
Also, make this `public` or even `open`? Besides the `x-grpc-web` header, this looks fairly non-GRPC-specific as well; should we drop the `GRPC` from the name?

Reply:
done